addons/project/static/src/js/project_form.js

```diff
@@ -125,9 +126,25 @@ export const FormDescriptionExpanderView = FormView.extend({
     }),
 })
 
+export const ProjectFormRenderer = FormDescriptionExpanderRenderer.extend({
+
+    /**
+     * @private
+     * @override
+     */
+    _renderStatButton: function (node) {
+        var $button = this._super.apply(this, arguments);
+        if ($button.attr('id') == 'button_open_parent_task' && this.state.data.parent_id) {
```

Comment:
Why not using the `name` attribute instead of the `id`? 🤔

Reply:
Done